tomcat/src/main/java/org/apache/catalina/connector/Connector.java

```diff
@@ -65,8 +74,8 @@ private void process(final Socket connection) {
             return;
         }
         log.info("connect host: {}, port: {}", connection.getInetAddress(), connection.getPort());
-        var processor = new Http11Processor(connection);
-        new Thread(processor).start();
+        var processor = new Http11Processor(connection, controllerContainer);
+        executorService.execute(processor);
```

Comment:
```
A main difference between the `submit()` and `execute()` method is that `submit()` can return the result of computation because it has a return type of `Future`, but the `execute()` method cannot return anything because's return type is void.
```

submit은 `Future` 타입의 계산 결과 값은 반환하지만 excute는 아무 값도 반환하지 않는군요! 그럼 위 상황에서는 excute를 쓰는 게 맞겠네요!

다들 `excute()` 쓰는 것 같길래 썼는데 덕분에 또 알아갑니다. 최고최고👍